src/Annotation/DataParserForm.php

```diff
@@ -5,13 +5,13 @@
 use Drupal\Component\Annotation\Plugin;
 
 /**
- * Defines an authentication annotation object.
+ * Defines a data parser annotation object.
```

Comment:
These annotations are old and  deprecated and should be removed since they are covered in `MigrateForm`